src/main/kotlin/SpeakerFetcher.kt

```diff
@@ -0,0 +1,127 @@
+import com.github.kittinunf.fuel.httpGet
+import com.github.kittinunf.result.Result
+import com.google.gson.GsonBuilder
+import com.squareup.moshi.Types
+import org.apache.commons.codec.digest.DigestUtils
+import org.apache.commons.lang3.StringUtils
+import org.jsoup.Jsoup
+import java.io.File
+import java.io.InputStream
+
+data class XebiaWhoIsSpeaker(var file: String, var name: String, var imageUrl: String)
+data class XebiaBlogSpeaker(var firstName: String, var lastName: String, var bio: String)
+
+object SpeakerFetcher {
+    lateinit var localUsers: List<XebiaWhoIsSpeaker>
```

Comment:
Not good practise to use `lateinit` except for `@Inject` u may handle nullability in your code